packages/archive/store-engine-sqleet/src/SQLeetEngine.test.ts

```
@@ -148,7 +148,6 @@ describe('SQLeetEngine', () => {
       try {
         const entity = {'name\'"`': 'Otto'};
         await engine.create('users', '1', entity);
-        fail();
```

Why?!

---

Good catch. Fail doesn't exist in jest, but I need to throw an error here